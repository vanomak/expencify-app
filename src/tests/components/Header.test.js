import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/header';

test('should render header correctly', () => {
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header/>);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
    //console.log(renderer.getRenderOutput());
    const wrapper = shallow(<Header/>);
    expect(wrapper).toMatchSnapshot();
});

