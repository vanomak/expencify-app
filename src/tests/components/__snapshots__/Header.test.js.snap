// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Expencify
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Create
  </NavLink>
  <button
    onClick={[Function]}
  >
    Log out
  </button>
</header>
`;
